### PR TITLE
ARN 934 - UPW availability for work accessibility rework

### PR DESCRIPTION
**Pull request by LucasCairns**
I've updated the checkbox labels on the UPW Availability page to provide additional context when navigating with a screenreader such as Voiceover. I've made the changes in the Wiremock stubs to test this out and will reflect the changes in the API if we're happy with the implmentation

For example, when navigating the checkboxes with a screenreader the user will be presented with
`Available for work Wednesday evening, ticked, checkbox` when ticked, or `Available for work Wednesday evening, unticked, checkbox` when unticked

This varies from the content described in the ticket, but can't see how we could implement those changes without reverting to Javascript as the checkboxes use a single boolean value 🤔

### CODE REVIEW

File: app/upw/templates/placement-details/availability.njk [hunk 1]
```diff
@@ -56,71 +56,72 @@
...
+              <legend id="individual_availability-legend" class="govuk-fieldset__legend individual_availability govuk-label--m">
+                {{ questions['individual_availability'].questionText }}
+              </legend>
+            </caption>
             <thead class="govuk-table__head">
-            <tr>
-              <th scope="row" class="govuk-table__header"></th>
-              <th scope="col" class="govuk-table__header upw-availability-table__column upw-availability-table__header">
+              <tr>
+                <td scope="row" class="govuk-table__header"></td>
```
Comment (LucasCairns) [inline app/upw/templates/placement-details/availability.njk:68]: WAVE was flagging the empty `th` so I've changed this to a `td`

File: app/upw/templates/placement-details/availability.njk [hunk 2]
```diff
@@ -56,71 +56,72 @@
...
         <input type="hidden" name="x-csrf-token" value="{{ csrfToken }}">
 
-        <fieldset class="govuk-fieldset" aria-describedby="individual_availability-hint">
-          <legend class="govuk-fieldset__legend individual_availability govuk-label--m">
-            {{ questions['individual_availability'].questionText }}
-          </legend>
-
+        <fieldset class="govuk-fieldset" aria-describedby="individual_availability-legend">
           <table class="govuk-table upw-availability-table">
+            <caption class="govuk-visually-hidden">
```
Comment (LucasCairns) [inline app/upw/templates/placement-details/availability.njk:61]: I've added the `caption` here do that the `table` is clearly identifiable when navigating using Voiceover

**Comment by Coupey** [comment]
> This varies from the content described in the ticket, but can't see how we could implement those changes without reverting to Javascript as the checkboxes use a single boolean value 🤔

I think that's absolutely fine - until getting into the details it was always going to be hard to specify exactly what the behaviour/text should be.

**Review by Coupey — APPROVED** [review]
LGTM

**Pull request merged**